jenkinsfiles/ginkgo-kernel.Jenkinsfile

```diff
@@ -0,0 +1,166 @@
+@Library('cilium') _
```

Comment:
Perhaps the filename should be `ginkgo-kernel-4-19`

Reply:
I just updated the jenkinsfile, this file is parametrized, it runs 4.19 by default, but can run any other future custom kernel versions that we support